storm-client/src/jvm/org/apache/storm/Constants.java

```diff
@@ -56,5 +56,7 @@
     public static final String STORM_ACTIVE_ATOM = "storm-active-atom";
     public static final String COMPONENT_TO_DEBUG_ATOM = "storm-component->debug-atom";
     public static final Object LOAD_MAPPING = "load-mapping";
+    public static final String USERCACHE = "usercache";
+    public static final String DEFAULT_BLOB_VERSION_SUFFIX = ".version";
```

Comment:
I'm not sure these needed to be moved.  They look like they were moved for one possible way of doing this, and then not reverted when they were not used by the client.